openshift/deployment.yaml

```diff
@@ -8,30 +8,30 @@ parameters:
   - name: TAG
     value: latest
     description: The output image tag beig deployed
-#  - name: DOCKER_REGISTRY
-#    required: true
+  - name: DOCKER_REGISTRY
```

Comment:
Are we using docker as a stop-gap?

Reply:
That's just a parameter key and the value is being passed to it through Jenkins file.